addons/config/templates/run.tanzu.vmware.com/v1alpha3/kappcontrollerconfig.yaml

```diff
@@ -20,6 +20,8 @@ spec:
       tolerations:
         - key: CriticalAddonsOnly
           operator: Exists
+        - effect: NoSchedule
+          key: node-role.kubernetes.io/control-plane
         - effect: NoSchedule
           key: node-role.kubernetes.io/master
         - effect: NoSchedule
```

Comment:
Could we also update https://github.com/vmware-tanzu/tanzu-framework/blob/main/apis/run/v1alpha3/kappcontrollerconfig_types.go#L60

Reply:
Updated